### PR TITLE
Add new bcrypt output to quantum password

### PR DESCRIPTION
**Pull request by JHeilCoveo**
This should not force a recreation of quantum_password

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/coveo/terraform-provider-quantum
+
+go 1.13
+
+require (
+	github.com/hashicorp/terraform v0.12.18
```
Comment (mouellet-coveo) [inline go.mod]: require terraform 0.12? Can this cause any versions issues?

Comment (JHeilCoveo) [reply go.mod]: I'm not sure but the provider builds and works with terraform 11.x

**Comment by JHeilCoveo** [comment]
Before this PR we had the following pattern.
`
resource "quantum_password" "admin_password" {
  special_chars = "!#"
}

data "template_file" "internal_users" {
  template = "${file("${path.module}/files/internal_users.yml")}"

  vars = {
    admin_hash               = "${bcrypt(quantum_password.admin_password.password)}"
  }
}
`
The content of internal_users would change on every run.

Now we have this and the content won't change anymore.
`
resource "quantum_password" "admin_password" {
  special_chars = "!#"
}
data "template_file" "internal_users" {
  template = "${file("${path.module}/files/internal_users.yml")}"

  vars = {
    admin_hash               = "${quantum_password.admin_password.bcrypt}"
  }
}
`

**Comment by tdegiacinto** [comment]
A Unit test for this resource would be great

**Pull request merged**